### PR TITLE
Add NOWNodes to Showcase

### PR DESCRIPTION
**Pull request by AnnaNodes**
## Checklist

- [x] I have read the [How to Contribute](https://developers.cardano.org/docs/portal-contribute/).
- [x] I have read the [Showcase Requirements](https://github.com/cardano-foundation/developer-portal/edit/staging/src/data/showcases.js)
- [x] I have run `yarn build` after adding my changes **without getting any errors**. 

## Showcase addition

* Title: NOWNodes
* Description: Get instant access to Cardano RPC nodes, indexer, and WebSocket connection. Start seamless integration on a FREE tariff plan. You can focus on growing your web3 products with us.
* Website:   https://nownodes.io
* Source: https://nownodes.io/nodes/cardano-ada
* Tags:
  * `daotool`
  * `explorer`

### CODE REVIEW

**Comment by rphair** [comment]
Some review already in https://github.com/cardano-foundation/developer-portal/pull/1148.

File: src/data/showcases.js [hunk 1]
```diff
@@ -1198,6 +1198,14 @@ export const Showcases = [
     source: null,
     tags: ["wallet", "nftsupport"],
   },
+  {
+    title: "NOWNodes",
+    description: "Get instant access to Cardano RPC nodes, indexer, and WebSocket connection. Start seamless integration on a FREE tariff plan. You can focus on growing your web3 products with us.",
```
Comment (rphair) [inline src/data/showcases.js:1203]: The marketing language here is unacceptable.  You can compare with other Dev Portal listings to rewrite this with the correct tone: to simply _include lists of features_ (emphasising any _distinctive_ features) without incentives or coercion.  If you need help doing this please post here and I'll make some recommendations.

File: src/data/showcases.js [hunk 2]
```diff
@@ -1198,6 +1198,14 @@ export const Showcases = [
...
+    preview: require("./showcase/nownodes.png"),
+    website: "https://nownodes.io",
+    source: null,
+    tags: ["daotool", "explorer"],
```
Comment (rphair) [inline src/data/showcases.js:1207]:
```suggestion
    tags: ["explorer"],
```
I don't see how this fits the `daotool` category so @AnnaNodes perhaps you could explain here why you included it: or merge this suggestion to remove this keyword.

**Review by rphair — COMMENTED** [review]
The inclusion of Cardano among the many blockchains supported by this service seems relatively insignificant, so likewise I don't see how this service is significant to Cardano.  If you offer some distinctive support of Cardano then @AnnaNodes please post a Cardano specific link here and/or include it on your listing to better support your request.

**Comment by AnnaNodes** [comment]
Thank you! Fixed everything with your suggestions. Hope it looks appropriate now.

File: src/data/showcases.js (hunk 1 above)
Comment (rphair) [reply src/data/showcases.js]: thank you @AnnaNodes for https://github.com/cardano-foundation/developer-portal/pull/1149/commits/9193fc57a9864a5110d710fcef55843683dc9a93 - the style is very appropriate now.

**Review by rphair — COMMENTED** [review]
Thanks for cleaning up the submission @AnnaNodes - personally I don't think the Dev Portal needs another node provider, especially given @rdlrt's observations about the inadequate free tier pointed out in https://github.com/cardano-foundation/developer-portal/pull/1148#issuecomment-1723387395, but I'll follow this listing & support it if any other reviewer makes a case otherwise.

**Comment by katomm** [comment]
The pull request has not received the required approvals and is closed. However, it can be opened again. Details on: https://developers.cardano.org/docs/portal-contribute/#how-are-pull-requests-reviewed

In any case, the discussion can be continued here.

**Pull request closed without merging**